docs/components/receive.md

```diff
@@ -12,6 +12,12 @@ For more information please check out [initial design proposal](../proposals-don
 
 > NOTE: As the block producer it's important to set correct "external labels" that will identify data block across Thanos clusters. See [external labels](../storage.md#external-labels) docs for details.
 
+As of version [v0.22.0](https://github.com/thanos-io/thanos/blob/main/CHANGELOG.md#added), the Thanos receiver implements the [receiver split proposal](../proposals-done/202012-receive-split.md), which allows data ingestion as well as request routing to be independently enabled/disabled. Thanks to these changes only routing-receivers need to watch for configuration changes, and ingesting-receivers do not need to. So when the configuration is updated, the routing-receivers will almost instantaneously re-configure without a lengthy WAL flush (& downtime) like we have in the current setup (routing & ingesting receivers). This would also enable users to prepare a topology of receivers, containing trees of receiver with depth **N**.
```

Comment:
How about this?
```suggestion
Since the [receiver split proposal](../proposals-done/202012-receive-split.md) has been implemented in thanos receive, this allows data ingestion as well as request routing to be independently enabled/disabled. Thanks to these changes only routing-receivers need to watch for configuration changes, and ingesting-receivers do not need to. So when the configuration is updated, the routing-receivers will almost instantaneously re-configure without a lengthy WAL flush (& downtime) like we have in the current setup (routing & ingesting receivers). This would also enable users to prepare a topology of receivers, containing trees of receiver with depth **N**.
```

Reply:
@squat could you please take a second look at this pr :) ?